src/03-modality-agnostic-files.md

```diff
@@ -81,29 +85,105 @@ Template:
 ```Text
 participants.tsv
 participants.json
-phenotype/<measurement_tool_name>.tsv
-phenotype/<measurement_tool_name>.json
 ```
 
-Optional: Yes
+The purpose of this RECOMMENDED file is to describe properties of participants
+such as age, sex, handedness etc.
+In case of single session studies this file has one compulsory column
+`participant_id` that consists of `sub-<label>`, followed by a list of optional
+columns describing participants.
+Each participant MUST be described by one and only one row.
+
+Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`,
+and `handedness`. We RECOMMEND to use the following values for these three
+columns in case that you use them:
```

Comment:
```suggestion
Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`,
and `handedness`. We RECOMMEND to make use of these columns, and
in case that you do use them, we RECOMMEND to use the following values
for them:
```